components/database/mongodb-40/Makefile

```diff
@@ -0,0 +1,129 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Alexander Pyhalov
+# Copyright 2018 Michal Nowak
+#
```

Comment:
I think your copyright should go here. Same for the rest of the files you edited.